packages/javascript/bh-shared-ui/src/components/Navigation/MainNav.test.tsx

```diff
@@ -145,41 +145,61 @@ describe('MainNav', () => {
     });
     it('should render a label and version number when expanded', async () => {
         const MainNavBar = await screen.findByRole('navigation');
-        const versionNumberContainer = await screen.findByTestId('main-nav-version-number');
-        const versionNumberLabel = await within(versionNumberContainer).findByText(/bloodhound/i);
+        expect(MainNavBar).toHaveClass('group');
+
+        const versionNumberContainer = await within(MainNavBar).findByTestId('main-nav-version-number');
         const versionNumberDigits = await within(versionNumberContainer).findByText(currentVersionNumber);
+        const versionNumberLabel = await within(versionNumberContainer).findByText(/bloodhound/i);
 
-        await user.hover(MainNavBar);
+        // ---- collapsed classes ----
+        expect(versionNumberDigits).toHaveClass('group-[:not(:hover)]:max-w-9');
+        expect(versionNumberLabel).toHaveClass('hidden');
+        expect(versionNumberLabel).toHaveClass('opacity-0');
+        // ---- collapsed classes ----
 
-        expect(versionNumberDigits).toBeInTheDocument();
-        expect(versionNumberLabel).toHaveClass('block');
+        // ---- classes displayed on hover ----
+        expect(versionNumberLabel).toHaveClass('group-hover:opacity-100');
+        expect(versionNumberLabel).toHaveClass('group-hover:block');
+        // ---- classes displayed on hover ----
```

Comment:
this ensures the child has the correct classes to display on hover of the parent